representative/models.py

```diff
@@ -1327,13 +1336,14 @@ def update_representative_social_media(self, representative, representative_twit
         }
         return results
 
-    def update_representative_twitter_details(self, representative, twitter_dict,
-                                                cached_twitter_profile_image_url_https,
-                                                cached_twitter_profile_background_image_url_https,
-                                                cached_twitter_profile_banner_url_https,
-                                                we_vote_hosted_profile_image_url_large,
-                                                we_vote_hosted_profile_image_url_medium,
-                                                we_vote_hosted_profile_image_url_tiny):
+    @staticmethod
+    def update_representative_twitter_details(representative, twitter_dict,
```

Comment:
If all variables can't fit on one line, please break each one onto it's own line.